src/main.rs

```diff
@@ -1,3 +1,40 @@
+extern crate clap;
+extern crate ignore;
+
+use clap::{App, Arg};
+use ignore::Walk;
+
 fn main() {
-    println!("Hello, world!");
+    let matches = App::new("fr")
+        .version("0.1.0")
+        .author("The ruplacer team")
+        .about("Find and replace")
+        .arg(Arg::with_name("PATERN")
+            .help("the pattern to grep for")
+            .required(true)
+            .index(1))
+        .arg(Arg::with_name("DIR")
+            .short("d")
+            .long("dir")
+            .value_name("DIR")
+            .index(2)
+            .help("the directory to find into")
+            .takes_value(true))
+        .arg(Arg::with_name("v")
+            .short("v")
```

Comment:
can't we have a --verbose with short -v?

Reply:
done